polus-imagej-filter-dog-plugin/src/main.py

```diff
@@ -0,0 +1,250 @@
+                out = ij.op().filter().dog(in1,sigmas1,sigmas2)
+            
+            logger.info('Completed op!')
+            if in1_path != None:
+                in1_br.close()
+            
+            # Saving output file to out
+            logger.info('Saving...')
+            out_array = ij_converter.from_java(ij, out,out_types[_opName])
+            bw = BioWriter(_out.joinpath(fname),metadata=metadata)
+            bw.Z = 1
+            bw.dtype = out_array.dtype
+            bw[:] = out_array.astype(bw.dtype)
+            bw.close()
+            
+    except:
+        logger.error('There was an error, shutting down jvm before raising...')
+        raise
+            
+    finally:
+        # Exit the program
+        logger.info('Shutting down jvm...')
+        del ij
+        jpype.shutdownJVM()
+        logger.info('Complete!')
+
+if __name__=="__main__":
+
+    ''' Setup Command Line Arguments '''
+    logger.info("Parsing arguments...")
+    parser = argparse.ArgumentParser(prog='main', description='Difference of Gaussians (DoG) implementation')
+    
+    # Add command-line argument for each of the input arguments
+    parser.add_argument('--opName', dest='opName', type=str,
+                        help='Operation to peform', required=True)
+    parser.add_argument('--inpDir', dest='in1', type=str,
+                        help='Input image collection to be processed by this plugin', required=True)
+    parser.add_argument('--sigma1', dest='sigma1', type=str,
+                        help='sigma1', required=False)
+    parser.add_argument('--sigma2', dest='sigma2', type=str,
+                        help='sigma2', required=False)
+    parser.add_argument('--sigmas1', dest='sigmas1', type=str,
+                        help='sigmas1', required=False)
+    parser.add_argument('--sigmas2', dest='sigmas2', type=str,
+                        help='sigmas2', required=False)
+    
+    # Add command-line argument for each of the output arguments
+    parser.add_argument('--outDir', dest='out', type=str,
+                        help='Output collection', required=True)
+    
+    """ Parse the arguments """
+    args = parser.parse_args()
+    
+    # Input Args
+    _opName = args.opName
+    logger.info('opName = {}'.format(_opName))
+    
+    _in1 = Path(args.in1)
+    logger.info('inpDir = {}'.format(_in1))
+    
+    _sigma1 = args.sigma1
+    logger.info('sigma1 = {}'.format(_sigma1))
+    
+    _sigma2 = args.sigma2
+    logger.info('sigma2 = {}'.format(_sigma2))
+    
+    _sigmas1 = args.sigmas1
+    logger.info('sigmas1 = {}'.format(_sigmas1))
+    
+    _sigmas2 = args.sigmas2
+    logger.info('sigmas2 = {}'.format(_sigmas2))
+    
+    # Output Args
+    _out = Path(args.out)
+    logger.info('outDir = {}'.format(_out))
+    
+    main(_opName=_opName,
+         _in1=_in1,
+         _sigma1=_sigma1,
+         _sigma2=_sigma2,
+         _sigmas1=_sigmas1,
+         _sigmas2=_sigmas2,
+         _out=_out)
```

Comment:
Was black run on this? The formatting doesn't seem correct.

Reply:
This is an older version using the old pipeline, black has been run on the updated version.